runtimes/polimec/src/weights/pallet_scheduler.rs

```diff
@@ -187,4 +187,11 @@ impl<T: frame_system::Config> pallet_scheduler::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+
+	// TODO: add benches
+	fn schedule_retry(_: u32) -> cumulus_primitives_core::Weight { Weight::zero() }
+	fn set_retry() -> cumulus_primitives_core::Weight { Weight::zero() }
+	fn set_retry_named() -> cumulus_primitives_core::Weight { Weight::zero() }
+	fn cancel_retry() -> cumulus_primitives_core::Weight { Weight::zero() }
+	fn cancel_retry_named() -> cumulus_primitives_core::Weight { Weight::zero() }
```

Comment:
Also Todo?

Reply:
This will come after we rerun benchmarks